account_avatax/models/account_move.py

```diff
@@ -226,6 +235,9 @@ def _avatax_compute_tax(self, commit=False):
             avatax_config.commit_transaction(self.name, doc_type)
             return tax_result
 
+        if self.avatax_amt_line_override and self.move_type == "out_refund":
```

Comment:
Can you confirm that the Invoice total tax amount is correct when you have some lines with Odoo TAx and other lines with Avalara tax?